hustle/jobs/templates/jobs/view_job.html

```diff
@@ -144,24 +144,29 @@ <h5>Submit A New Bid</h5>
             </div>
```

Comment:
Should have a Complete Survey button next to the Mark Complete button for workers